hasura.planx.uk/tests/teams.test.js

```diff
@@ -1,20 +1,18 @@
 const { introspectAs } = require("./utils");
 
-describe("teams and team_members", () => {
```

Comment:
i understand we're not populating table yet so no expection this happens as this PR, but let's make sure introspection tests are added back for new table sooner than later - would we expect public versus admin privileges to work same as before here still?

Reply:
I'll add these back in a quick separate PR just to avoid a few rebases if that's ok?